packages/devtools_app/lib/src/shared/analytics/_analytics_web.dart

```diff
@@ -113,72 +109,39 @@ class GtagEventDevTools extends GtagEvent {
     int? inspector_tree_controller_id, // metric12
   });
 
-  @override
```

Comment:
Totally up to you, but one of the nice things about implementing another extension type is you get their members, so these could be omitted since `GtagEvent` already declares them.